packages/screenshot/android/src/main/java/io/capawesome/capacitorjs/plugins/screenshot/Screenshot.java

```diff
@@ -1,11 +1,34 @@
 package io.capawesome.capacitorjs.plugins.screenshot;
 
-import android.util.Log;
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.webkit.WebView;
+import androidx.annotation.NonNull;
+import io.capawesome.capacitorjs.plugins.screenshot.classes.interfaces.NonEmptyCallback;
+import io.capawesome.capacitorjs.plugins.screenshot.classes.interfaces.Result;
+import io.capawesome.capacitorjs.plugins.screenshot.classes.results.TakeResult;
+import java.io.File;
+import java.io.FileOutputStream;
 
 public class Screenshot {
 
-    public String echo(String value) {
-        Log.i("Echo", value);
-        return value;
+    private final WebView webView;
+
+    public Screenshot(WebView webView) {
+        this.webView = webView;
+    }
+
+    public void take(Context context, @NonNull NonEmptyCallback<Result> callback) {
```

Comment:
```suggestion
    private final ScreenshotPlugin plugin;

    public Screenshot(@NonNull ScreenshotPlugin plugin) {
        this.plugin= plugin;
    }

    public void take(@NonNull NonEmptyCallback<Result> callback) {
```

Please pass the plugin instance and remove the context param, see https://github.com/capawesome-team/capacitor-plugins/blob/b5088c73cef1e2be5d915d1ed7a0649865a3d3ca/packages/live-update/android/src/main/java/io/capawesome/capacitorjs/plugins/liveupdate/LiveUpdate.java#L92